data/countries_names.txt

```diff
@@ -45271,39 +45271,39 @@
   zh-Hans = 斯德哥尔摩
   zh-Hant = 斯德哥爾摩
 
-[Switzerland_Espace Mittelland_Bern]
```

Comment:
Yeap this is an internal name of a map / mwm file. Its not user visible and is more about how a country is divided into several countries (which is complicated and not easy to change). 

#9759 gets it right.